examples/demo/config/development.yaml

```diff
@@ -109,7 +112,7 @@ database:
 # Redis Configuration
 redis:
   # Redis connection URI
-  uri: {{ get_env(name="REDIS_URL", default="redis://127.0.0.1") }}
+  uri: { { get_env(name="REDIS_URL", default="redis://127.0.0.1") } }
```

Comment:
note auto formatting breaks the templating format, it should be `{{`. try "save without formatting" for YAML

Reply:
Fixed it, seems like `JetBrains` treats `{{` as two `{` and not as a special template, had to manually fix it (no formatting option support it :( )